src/makeRobotAccountant.js

```diff
@@ -6,7 +6,29 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let times = 0;
+
+  return function getSum(a) {
+    function x(b) {
+      let sum = 0;
+
+      sum += a + b;
+
+      if (times <= 3) {
+        return sum;
+      }
+
+      if (times % 2 !== 0 && times >= 5) {
+        return sum;
```

Comment:
Here is the same returned value in both cases. Combine them. And then you'll be able you use a ternary operator btw